src/components/twilio-voice-dialer/public/index.js

```diff
@@ -1 +1,10 @@
-// TODO: Implement example usage of the Web Component
+(async function () {
+  const identity = '';
+  const response = await fetch(
+    `http://127.0.0.1:3030/token?identity=${identity}`
```

Comment:
Do we want to use `localhost` as opposed to `127.0.0.1`? There might be implications around IPv4 vs IPv6 that favor `localhost` but I'm not knowledgable enough about the subject matter to give a concrete answer.

Reply:
This is just temporary until @charliesantos finishes the server implementation